packages/react-instantsearch-hooks-dom/src/widgets/ClearRefinements.tsx

```diff
@@ -34,7 +35,10 @@ export function ClearRefinements(props: ClearRefinementsProps) {
   return (
     <ClearRefinementsUiComponent
       {...otherProps}
-      resetLabel={resetLabel}
+      translations={{
+        resetLabel: 'Clear refinements',
+        ...translations,
```

Comment:
This assumes translations are all single-level objects, which I believe is easier to grasp for users anyway.